### PR TITLE
Add API for detecting language

### PR DESCRIPTION
**Pull request by mkozakov**
```golang
res, err := co.DetectLanguage(DetectLanguageOptions{
	Texts: []string{"this text is in english", "Этот текст на Русском языке."},
})

fmt.Print(res.Results[0].LanguageCode) // "en"
fmt.Print(res.Results[0].LanguageName) // "English"

fmt.Print(res.Results[1].LanguageCode) // "ru"
fmt.Print(res.Results[1].LanguageName) // "Russian"
```

### CODE REVIEW

File: detectlanguage_test.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	co, err := CreateClient(apiKey)
+	if err != nil {
+		t.Error(err)
+	}
+
+	t.Run("DetectLanguage success", func(t *testing.T) {
+		res, err := co.DetectLanguage(DetectLanguageOptions{
+			Texts: []string{"this text is in english", "Этот текст на Русском языке."},
+		})
+		assert.Nil(t, err)
```
Comment (abdullahkady) [inline detectlanguage_test.go:19]: shouldn't we assert language name too? and maybe confidence (tricky since we don't mock of course, but at least that there exists a float > 0).

File: detectlanguage_test.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDetectLanguage(t *testing.T) {
+	co, err := CreateClient(apiKey)
+	if err != nil {
+		t.Error(err)
+	}
+
+	t.Run("DetectLanguage success", func(t *testing.T) {
```
Comment (abdullahkady) [inline detectlanguage_test.go:15]: I think we need to add an invalid request test too, maybe something basic like an empty string

Comment (mkozakov) [reply detectlanguage_test.go:15]: We are not doing any special error handling, so I don't think there is anything to test with that

**Review by abdullahkady — APPROVED** [review]
